action/lint.go

```diff
@@ -0,0 +1,160 @@
+package action
+
+import (
+	"github.com/google/go-github/github"
+	"github.com/helm/helm/chart"
+	"github.com/helm/helm/log"
+	"github.com/helm/helm/manifest"
+	"gopkg.in/yaml.v2"
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+)
```

Comment:
Is there an order for go imports that we should follow? I've seen something like the order of:

- golang builtin libraries
- imported libraries
- internal (to the current project) libraries